src/options.zig

```diff
@@ -2572,7 +2575,7 @@ pub const PathTemplate = struct {
     };
 
     pub const file = PathTemplate{
-        .data = "[dir]/[name].[ext]",
+        .data = "[dir]/[name].[target].[ext]",
```

Comment:
TODO: before merging, undo this and have two separate templates that get swapped out automatically if unset: one when using this and one when not